Componets/Image/image.css

```diff
@@ -0,0 +1,18 @@
+.image-container{
+    max-width: 40vw;
+    margin: 2.4rem auto;
+}
+.img-responsive{
+    max-width: 100%;
+    object-fit: cover;
+    height: auto;
+}
+.img-square{
+    width: 200px;
```

Comment:
Don't use absolute units if you plan to make it responsive, try to switch on relative units (probably rem).

Reply:
I have created standard size image of 200px * 200px that's why given units in px .